cron/src/tests/test_message_service.py

```diff
@@ -1,49 +1,43 @@
 import logging
 from datetime import datetime
-from pytest import fixture
+import pytest
 from freezegun import freeze_time
 from notifications.message_service import MessageService
 from notifications.definitions import RecordInfo, BookInfo, BorrowerInfo
 
 
 class TestMessageService():
-    def test_compose_messages_senders(self, test_data):
+    @pytest.mark.parametrize(
+        'test_method, lenght',
+        [('compose_user_messages', 2), ('compose_admin_messages', 1)]
+    )
+    def test_compose_messages_senders(self, test_data, test_method, lenght):
```

Comment:
It should be 'recipients' instead of 'senders' in test name as it tests if recipient in message body is correct.